planning_period: 0.01 # seconds
world_frame: "local_origin"
# path_offset: 2.5 # metres
approach_speed: 1.0 # metres per second
trajectory:
  sampling_dt: 0.2 # seconds
  horizon: 8.0 # seconds

z_bounds:
  min: 0.0
  max: 20.0
